quipucords/scanner/network/connect_callback.py

```diff
@@ -40,6 +41,7 @@ def __init__(self, result_store, credential, display=None):
 
     def v2_runner_on_ok(self, result):
         """Print a json representation of the result."""
+        input_log.log_ansible_result(result)
```

Comment:
May want to try/catch around this so any IO or other related failure in this flow doesn't kill a scan.

Reply:
Done, via a helper function.